contrib/win32/openssh/appveyor.psm1

```diff
@@ -494,43 +500,15 @@ function Deploy-OpenSSHTests
     }
     
 
-    [System.IO.DirectoryInfo] $repositoryRoot = Get-RepositoryRoot    
+    [System.IO.DirectoryInfo] $repositoryRoot = Get-RepositoryRoot
     
     $sourceDir = Join-Path $repositoryRoot.FullName -ChildPath "regress\pesterTests"
-    Copy-Item -Path "$sourceDir\*" -Destination $OpenSSHTestDir -Include *.ps1,*.psm1 -Force -ErrorAction Stop
+    Copy-Item -Path "$sourceDir\*" -Destination $OpenSSHTestDir -Include *.ps1,*.psm1, sshd_config -Force -ErrorAction Stop
 
-    $sourceDir = Join-Path $repositoryRoot.FullName -ChildPath "bin\$folderName\$RealConfiguration"    
+    $sourceDir = Join-Path $repositoryRoot.FullName -ChildPath "bin\$folderName\$RealConfiguration"
     Copy-Item -Path "$sourceDir\*" -Destination $OpenSSHTestDir -Exclude ssh-agent.exe, sshd.exe -Force -ErrorAction Stop
```

Comment:
This is Deploy-OpenSSHTests. the purpose is to deploy unit tests. but * wildcard includes all files (when develop this script, the name pattern of the unit test files are not unique).  ssh-agent.exe, sshd.exe are excluded because the service is already started and there will be error if deploy again. I will change the pattern ( to unittest-*) to only deploy unit tests to avoid confusion.